src/one_dragon/base/operation/one_dragon_context.py

```
@@ -297,11 +297,68 @@ def init(self) -> None:
+                
+                # 若命令返回码非 0 且 stderr 存在内容，才视为真实错误返回
+                stderr_msg = result.stderr.strip()
+                if result.returncode != 0 and stderr_msg:
+                    log.warning(f"获取当前 Git hooksPath 失败: {stderr_msg}")
+                    return
+                current_hooks_path = result.stdout.strip()
+                
+                # 如果没有指向 .githooks 则配置
+                if current_hooks_path != ".githooks":
+                    set_result = subprocess.run(
+                        ["git", "config", "core.hooksPath", ".githooks"],
```

Ruff (0.15.17)

[error] Starting a process with a partial executable path

(S607)